mycroft/skills/core.py

```diff
@@ -268,6 +260,9 @@ def bind(self, emitter):
                            self.handle_enable_intent)
             self.add_event('mycroft.skill.disable_intent',
                            self.handle_disable_intent)
+            n, f = 'mycroft.skills.settings.update', self.settings.run_poll
```

Comment:
Nice!  Are you intending to emit the "mycroft.skills.settings.update" from somewhere specific?  Seems like this should be generated as part of the "Update configuration" handler in skill-configure.

NOTE:  I'm personally not a huge fan of the
```python
n,f = something, something_else
```
notation instead of
```python
n = something
f = something_else
```

Reply:
Also, be sure to add the message to https://docs.google.com/spreadsheets/d/1RsBpMOGx4o8A_zF_UELYQSLK8DlIScHGnfzJ3iSN0AE/edit#gid=0

Reply:
Added to the doc, and you are correct about emitting it in skill-configuration. I have a local change that does that. Also, I was in between the two ways, but used the side by side method to make it analogous to `.emit(name, func)`

Reply:
I see the .emit(name, func) parallel, but I think that assignment construct is unique enough to Python that it could be a little confusing and it doesn't seem to add a ton of clarity.  Just my $0.02, not a huge deal.